### ISSUE TITLE
TypeError when opening tab 'Accounts->Configuration'

### ISSUE THREAD

**Issue by lufog** [getgrav/grav#3418]
```
TypeError Grav\Framework\Flex\FlexDirectory::loadDirectoryConfig(): Return value must be of type array, null returned
```
![image](https://user-images.githubusercontent.com/10379487/128353983-39fa56d2-a22a-4e9d-b596-bb2876cc4005.png)

OS: Windows 11
PHP: 8.0.8 ts x64
Server: `php -S 0.0.0.0 system\router.php`
Grav: 1.7.18 (clean instalation)
Admin plugin: 1.10.18
Flex Objects: 1.0.16

**Comment by zigojacko** [comment]
Getting exact same issue here.

**Comment by rhukster** [comment]
@zigojacko also on windows?

**Comment by zigojacko** [comment]
> @zigojacko also on windows?

No, the server hosting is Linux (CentOS 7).

**Comment by mahagr** [comment]
Your fix is broken, too. Did you take a look into my fix in the other issue (https://github.com/getgrav/grav/commit/7b1a188cfe59a54fc70d12f02b9477c71b7ccec4)?

I may have changed something else before that commit as the code looks different to what you have.

**Comment by kbftech** [comment]
@mahagr, you're right; my bad! The $path was never set because I was missing this line:
` $path = strtr(substr($uri, 9,  -5), '/', '.');`

I rectified my local code to match your fix and it seems to work fine. Thanks again for you help! 

NOTE: I will remove my snippets to prevent confusion.

**Comment by lufog** [comment]
This bug is fixed in Grav v1.7.19.
